### DISCUSSION TITLE
failing to deploy a contact on zksync

### DISCUSSION THREAD

**Discussion by didier-building** [Cyfrin/foundry-full-course-cu#2907] [General]
i'm trying to deploy a contact on zksync but i'm getting this error

Error: "underlying network changed (event=\"changed\", network={\"chainId\":300,\"name\":\"zkSync Sepolia Testnet\"}, detectedNetwork={\"name\":\"sepolia\",\"chainId\":11155111,\"ensAddress\":null}, code=NETWORK_ERROR, version=providers/5.7.2)"

**Comment by EngrPips** [comment]
Ensure your Metamask network is on `Zkync- sepolia` and not `Sepolia`

**Reply by chainflash** [reply]
great 👍
